istioctl/pkg/writer/pilot/status.go

```diff
@@ -103,5 +104,15 @@ func xdsStatus(sent, acked string) string {
 	if sent == acked {
 		return "SYNCED"
 	}
-	return "STALE"
+	timeSent, _ := parseTime(sent)
+	timeAcked, _ := parseTime(acked)
```

Comment:
if timeAcked == time.Time{} we should return something like "STALE (Not Acked)" otherwise you get a non obvious number e.g. `2562047h47m16.854775807s`.